packages/api/src/command/medical/document/document-query-storage-info.ts

```diff
@@ -35,8 +32,7 @@ type SimpleFile = {
 export function getDocToFileFunction(patient: Pick<Patient, "cxId" | "id">) {
   // TODO convert the input from CW Document to a Metriport shape
   return async (doc: Document): Promise<SimpleFile> => {
-    const extension = getFileExtension(doc.content?.mimeType);
```

Comment:
why are we removing the extension here?

Reply:
1. This was not the extension, but the mime type, so it was incorrect from the start.
2. We don't have the extension on the file name.

Reply:
That's true for upload only. @leite08 to review this and have both with the same pattern.